content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -41,14 +41,14 @@ be requested as well.
 apiVersion: v1
 kind: Pod
 metadata:
-  generateName: hugepages-volume-
+  name: hugepages_pod
```

Comment:
nit:
```suggestion
  name: huge-pages-example
```
`-` is more common than `_` in Pod names. Also the dash between huge and pages makes it a little more clear that this isn't a keyword.